Tests/Unit/CacheManagerTest.php

```diff
@@ -82,6 +82,7 @@ public function testRefreshRoute()
 
     public function testTagResponse()
     {
+        $this->markTestSkipped('TODO refactor to use tag handler');
```

Comment:
check after https://github.com/FriendsOfSymfony/FOSHttpCache/pull/237 is sorted out